docs/revsets.md

```diff
@@ -84,7 +84,7 @@ revsets (expressions) as arguments.
 * `children(x)`: Same as `x+`.
 * `ancestors(x)`: Same as `:x`.
 * `descendants(x)`: Same as `x:`.
-* `connected(x)`: Same as `x:x`.
+* `connected(x)`: Same as `x:x`. Useful when `x` includes several revisions.
```

Comment:
Should we use the terminology "commit" instead of "revision" here?

Reply:
Good point!